docs/metrics.md

```diff
@@ -12,6 +12,9 @@ All metrics documented here are auto-generated by the utility tool `tools/doc-ge
 ### kubevirt_info
 Version information.
 
+### kubevirt_abnormal_state
```

Comment:
@avlitman  This needs to be cnv_abnorml. Please move the recording rule to hco.

Reply:
you mean move to hco the cnv_abnormal only?

Reply:
Yes

Reply:
moved: https://github.com/kubevirt/hyperconverged-cluster-operator/pull/2855